build-system/tasks/visual-diff/index.js

```diff
@@ -22,15 +22,17 @@ const fs = require('fs');
 const gulp = require('gulp-help')(require('gulp'));
 const JSON5 = require('json5');
 const path = require('path');
-const puppeteer = require('puppeteer');
 const request = BBPromise.promisify(require('request'));
 const sleep = require('sleep-promise');
 const tryConnect = require('try-net-connect');
-const {execScriptAsync} = require('../../exec');
+const {execOrDie, execScriptAsync} = require('../../exec');
 const {gitBranchName, gitBranchPoint, gitCommitterEmail} = require('../../git');
 const {log, verifyCssElements} = require('./helpers');
 const {PercyAssetsLoader} = require('./percy-assets-loader');
-const {Percy} = require('@percy/puppeteer');
+
+// optional dependencies for local development (outside of visual diff tests)
+let puppeteer;
+let percy;
```

Comment:
Make this `Percy`, and later leave the `{Percy} = require('@percy/puppeteer')` will make it easier, instead of introducing the underscore to the arguments that get passed around

Reply:
Done!